tcms/core/widgets.py

```diff
@@ -44,7 +44,13 @@ def render(self, name, value, attrs=None, renderer=None):
             className: "fa fa-paperclip",
             title: "Attach File",
         },
-        "|", "preview", "side-by-side", "fullscreen", "|", "guide"]
+        "|", "preview", "side-by-side", "fullscreen", "|", "guide"
+    ],
+    autosave: {
+        enabled: true,
+        uniqueId: "MyUniqueID",
+        delay: 1000,
```

Comment:
Not sure if 1 sec isn't too much for some browsers. Let's default to 5 sec and see what people say about this.